Dockerfile

```diff
@@ -72,6 +72,7 @@ RUN mkdir -p /var/www/html \
         libxslt-dev \
         gd-dev \
         geoip-dev \
+        apache2-utils \
```

Comment:
The package `apache2-utils` is needed to compile only or it is needed after the compilation?

Reply:
> The package `apache2-utils` is needed to compile only or it is needed after the compilation?

Oh, sorry about that, I d't looking carefully just add to  deps

Reply:
> > The package `apache2-utils` is needed to compile only or it is needed after the compilation?
> 
> Oh, sorry about that, I d't looking carefully just add to  deps

Should be fixed now